src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -28,6 +28,7 @@ illegal.type=Usage of type ''{0}'' is not allowed.
 inline.conditional.avoid=Avoid inline conditionals.
 instantiation.avoid=Instantiation of {0} should be avoided.
 magic.number=''{0}'' is a magic number.
+matchxpath.match=Illegal code structure detected
```

Comment:
All other messages end with a period. This one (and all translations) should end with period too.

Reply:
done